articles/getting-to-know-css-blend-modes/demo_blendmodes_canvas.html

```diff
@@ -2,8 +2,8 @@
 <html>
 <head>
 <title>Blend modes with &lt;canvas&gt;</title>
-<script src="http://use.edgefonts.net/open-sans-condensed:n3,i3,n7:all.js"></script>
-<script src="http://use.edgefonts.net/paytone-one.js"></script>
+<script src="//use.edgefonts.net/open-sans-condensed:n3,i3,n7:all.js"></script>
```

Comment:
I wouldn’t call _universal_ solution “antipattern”, we live in the real world, not in the https-only